src/snyk/common/configuration/configuration.ts

```diff
@@ -204,13 +199,6 @@ export class Configuration implements IConfiguration {
     return `${hostnameParts[2]}.${hostnameParts[3]}`.includes('snykgov.io');
   }
 
-  get analyticsPermitted(): boolean {
```

Comment:
The code deleted in this file doesn't seem to be used anywhere

Reply:
This was added in https://github.com/snyk/vscode-extension/commit/8c2811b810f41c0507650a497266c54b15ae697d but the itly analytics have been removed since